guide/000-introduction.md

```diff
@@ -36,11 +58,12 @@ Telegram groups (by number of members):
 > 
 > * [Your first Telegram bot!](./001-your-first-bot.md)
 > * [Listening for updates and responding to them](./002-listen-and-respond.md)
+> * [Formatting messages](./formatting.md)
```

Comment:
These 3 were numbered `???` because it was not clear yet where in the guide order they should be, since there is missing content. If numbers are removed, they should be removed from every doc here.